src/services/utils.js

```diff
@@ -164,7 +174,10 @@ function isSpaceLogic(uri) {
  * @return {Element}
  */
 function getSpaceElFromLogic(prefix, logicEl) {
-  return dom.closest(logicEl, `[data-uri^="${prefix}/components/clay-space"]`);
+  var el = dom.closest(logicEl, `[data-uri^="${prefix}/components/clay-space"]`) ||
```

Comment:
is there a better way to write this? feels kinda hoke-y....

Reply:
I think that's actually the simplest ¯\\\_(ツ)_/¯

Reply:
the `matches` and `querySelectorAll` calls that `dom.closest` relies on supports comma-separated selectors, so this could be simplified to a single dom.closest call